functions/print_config.pp

```diff
@@ -0,0 +1,21 @@
+# @summary Print a configuration value if it is defined and the version is supported
+# @param name the config item name
+# @param value the config item value
+# @param version the version when the config item was introduced
+# @return the config item as a string or an empty string if the version is not supported
+function unbound::print_config (
+  String                                                        $name,
+  Optional[Variant[Boolean, Integer, String, Array[String, 1]]] $value   = undef,
+  Optional[String[1]]                                           $version = undef,
+) >> Optional[String] {
```

Comment:
Is there a situation where `undef` is returned? I think it's always a string?

Reply:
nice catch i think i started of retuning undef, but changed to '', fixed